docs/src/main/sphinx/release/release-382.md

```diff
@@ -0,0 +1,96 @@
+# Release 382 (25 May 2022)
+
+## General
+
+* Improve join performance. ({issue}`12257`)
+
+## Security
+
+## Web UI
+
+## JDBC driver
+
+## Server RPM
+
+## Docker image
+
+## CLI
+
+## Accumulo connector
+
+## Atop connector
+
+## BigQuery connector
+
+* Add support for [query results cache](https://cloud.google.com/bigquery/docs/cached-results).
+  This can be enabled using the `bigquery.query-results-cache.enabled` configuration property. ({issue}`12408`)
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+## Delta Lake connector
+
+* Support predicate pushdown for `timestamp with time zone` type. ({issue}`12007`)
+
+## Druid connector
+
+## Elasticsearch connector
+
+## Google Sheets connector
+
+## Hive connector
+
+* Fix failure when ACID table column name collides with ACID internal names. ({issue}`12401`)
```

Comment:
@ebyhr can you explain more .. as it stands I am not sure I can make sense of it

Reply:
Previously, query failed when ACID table contains ACID column names (e.g. `operation`)

Reply:
> you mean columns potentially used some sort of keywords as columns names and that would result in query failures? 

Yes. The column names are operation, originalTransaction, bucket, rowId, row & currentTransaction.

Reply:
If that's the whole list, I would just say:

> Fix query failure when a transactional table contains a column named `operation`, `originalTransaction`, `bucket`, `rowId`, `row`, or `currentTransaction`.